posix.mak

```diff
@@ -42,6 +42,9 @@ STABLE_DMD_CONF=$(STABLE_DMD).conf
 STABLE_RDMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))/rdmd \
 	--compiler=$(STABLE_DMD) -conf=$(STABLE_DMD_CONF)
 
+# whether to use a special theme for nightly changelog builds
+BUILD_NIGHTLY=1
```

Comment:
See also https://github.com/dlang/tools/pull/222 for an example on more "customization" based on `BUILD_NIGHTLY`